dnfdaemon-server/services/rpm/rpm.cpp

```diff
@@ -29,7 +29,7 @@ along with dnfdaemon-server.  If not, see <https://www.gnu.org/licenses/>.
 #include <libdnf/rpm/solv_query.hpp>
 #include <libdnf/rpm/transaction.hpp>
 #include <libdnf/transaction/transaction_item.hpp>
-#include "libdnf/utils/utils.hpp"
+#include <libdnf/utils/utils.hpp>
```

Comment:
Why do you use `<libdnf/utils/utils.hpp>` instead of `"libdnf/utils/utils.hpp"`? I suggest that we want to use `""` because it first search in project and not in additional include available paths (installed libdnf). See https://stackoverflow.com/questions/21593/what-is-the-difference-between-include-filename-and-include-filename. I belive that now there is no difference because you do not have libdbf-5 installed but in future, there could be a differences.

Reply:
Because this is not an issue of PR I am fine with an issue. Only revert the one line change because it is not the right direction.

Reply:
OK, this one occurrence is fixed and the rest is tracked as a Jira issue.